spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/EncryptionController.java

```diff
@@ -123,7 +123,7 @@ public String decrypt(@RequestBody String data, @RequestHeader("Content-Type") M
 	@PostMapping("/decrypt/{name}/{profiles}")
 	public String decrypt(@PathVariable String name, @PathVariable String profiles, @RequestBody String data,
 			@RequestHeader("Content-Type") MediaType type) {
-		TextEncryptor encryptor = getEncryptor(name, profiles, "");
+		TextEncryptor encryptor = getEncryptor(name, profiles, data);
```

Comment:
We don't even need this line, `encryptor` is not used in the `catch` block so we can just move the variable definition into the `try` block which already passes `data`.

Reply:
done in commit e7669e1b